src/surveys/components/ContactFormResponse.tsx

```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import { Input } from '../../components';
+import { FormField, FormSettings, Question, Theme } from '../../types';
+import { Button } from '../../components';
+
+type Props = {
+  question: Question;
+  theme?: Theme;
+  onAnswered: (questionId: string | number, answer: string) => void;
+};
+
+export const ContactFormResponse = ({ question, theme, onAnswered }: Props) => {
+  return (
+    <div className={'space-y-6'}>
+      <div>
+        {(question.options as FormField[]).map((option) => {
+          return (
+            <div className={'mb-3'}>
+              <Input
+                name={option.id + ''}
+                id={option.id}
+                label={option.label}
+                type={option.type}
+                onChange={() => {}}
+              />
+            </div>
+          );
+        })}
+      </div>
+      <Button
+        label={(question.settings as FormSettings).consentText}
```

Comment:
@afolasope The `consentText` here should be `submitText` in `SurveySettings`.

Reply:
@afolasope The `consent` and `disclaimer` are still missing in this component. If `consent` is true, we should add a checkbox after the last field with `consentText` as label and it should be `required`. If `disclaimer` is true, we should `disclaimerText` after `consent` checkbox and it should be just text but with a padlock icon.

[<img width="391" alt="Screenshot 2023-08-25 at 20 43 54" src="https://github.com/Formily/formily-web/assets/113894296/16645c66-d910-41af-8b4a-4524cd85862d">
](url)